src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -119,11 +126,19 @@ public Elevator() {
     gear11 = new DutyCycleEncoder(DIOConstants.gear11ID, 1.0, ElevatorConstants.gear11Offset);
     elevatorMotor1 = new TalonFX(CANIDs.kElevatorMotor1, Constants.RIOName);
     elevatorMotor2 = new TalonFX(CANIDs.kElevatorMotor2, Constants.RIOName);
+
+    elevatorConfig1 = new TalonFXConfiguration();
+    elevatorConfig2 = new TalonFXConfiguration();
+    elevatorConfig1.ClosedLoopRamps.VoltageClosedLoopRampPeriod = 0.02;
+    elevatorConfig2.ClosedLoopRamps.VoltageClosedLoopRampPeriod = 0.02;
+
+    elevatorMotor1.getConfigurator().apply(elevatorConfig1);
     elevatorMotor1.getConfigurator().apply(loopCfg);
     elevatorMotor1.getConfigurator().apply(elvSoftLimit);
     elevatorMotor1.getConfigurator().apply(currentLimit);
     elevatorMotor1.getConfigurator().apply(invertMotor);
     elevatorMotor1.getConfigurator().apply(elvMotionMagic);
+    elevatorMotor2.getConfigurator().apply(elevatorConfig2);
     elevatorMotor2.setControl(new Follower(CANIDs.kElevatorMotor1, true));
```

Comment:
@KianChilds menstioned that these should be put into one Config and applied once. This would be a good change to make in main